index.js

```diff
@@ -20,17 +21,44 @@ const register = (server, pluginOptions) => {
   }
   // set up prom metrics observers:
   const metric = {
+    methods: {
+      cache: {
+        hits: new prom.Counter({ name: 'cache_hits', help: 'cache hits', labelNames: ['method'] }),
```

Comment:
Let's change to hapi_method_cache instead of cache. 

We should probably change the other ones to hapi instead of http too